Makefile

```diff
@@ -22,7 +22,7 @@ DIRS := \
   contrib/broker/server \
   controller \
   pkg \
-  pkg/api
+  pkg/apis/servicecatalog
```

Comment:
Given https://github.com/kubernetes/kubernetes/tree/master/pkg/api
I think it should be pkg/api/servicecatalog